Realm/RLMObject.mm

```diff
@@ -153,3 +154,15 @@ + (NSArray *)ignoredProperties {
 }
 
 @end
+
+@implementation RLMDynamicObject
+
+- (id)valueForKey:(NSString *)key {
```

Comment:
This won't work for valid properties of RLMObject such as `realm` and `objectSchema`. It probably should. One way to do this cleanly is to instead implement `valueForUndefinedKey`